shared/message/planning/WalkPath.proto

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2023 NUbots <nubots@nubots.net>
+ */
+syntax = "proto3";
+
+package message.planning;
+
+import "Vector.proto";
+
+/// Plan a path to walk to the specified point
+message WalkTo {
+    /// The point to walk to, relative to the torso
+    fvec3 rPTt = 1;
+}
+
+/// Turn on the spot in the given direction
+message TurnOnSpot {
+    /// Direction to turn
+    bool clockwise = 2;
```

Comment:
I don't think it really matters but is there any reason the IDs for these two start at 2 and not 1?

Reply:
because I removed something and didn't notice the numbers :laughing: I'll change it